exla/test/exla/mlir/executable_test.exs

```diff
@@ -24,4 +24,24 @@ defmodule EXLA.MLIR.ExecutableTest do
     expected = Nx.add(t2, t2)
     assert_equal(result, expected)
   end
+
+  describe "binary ops" do
+    @bin_ops [:add, :subtract, :multiply, :pow, :min] ++
+               [:max, :remainder, :atan2]
+    # TO-DO (mlir): test for type casting
```

Comment:
Maybe set the defn_options for this process to be `compiler: EXLA, compiler_mode: :mlir` and then include the doctests from Nx one by one? The doctests define and jit on the fly, so it will be good to test this. :)